.github/workflows/release.yaml

```
@@ -21,14 +21,21 @@ jobs:
             curl \
             jq \
             squashfs-tools \
-            xz-utils
+            xz-utils \
+            gawk
+
+          KBS_VERS=$(curl -fsSL --retry-delay 1 --retry 60 --retry-connrefused --retry-max-time 60 --connect-timeout 20  https://raw.githubusercontent.com/kubernetes/website/main/data/releases/schedule.yaml | yq -r '.schedules[].previousPatches[0].release' | awk '{print "kubernetes-v"$1}')
+          [[ -z "${KBS_VERS}" ]] && exit 1
```

```suggestion
          if [[ -z "${KBS_VERS}" ]]; then; echo "Failed fetching Kubernetes versions"; exit 1; fi
```

---

Don't really like that if construct. I will make it a little different, but functional the same.
I hope you will agree. 
```suggestion
          [[ -z "${KBS_VERS}" ]] && echo "Failed fetching Kubernetes versions" && exit 1
```